.github/workflows/pr-testing-with-test-project.yml

```diff
@@ -9,22 +9,13 @@ jobs:
   test:
     if: github.event.pull_request.draft == false
     name: Test generator as dependency with Node ${{ matrix.node }}
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
Is there a reason why we dont use `matrix`?
```
    name: ${{ matrix.os }}
    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest, windows-latest]
```

Reply:
you mean why we do not test on all Os? I just did not want to extend testing time to much, this is why the other tests test on all OS, while these integration tests test only on Linux but different node versions

and only ubuntu, for now, has docker-compose preinstalled (but this is minor stuff)

Reply:
Fair enough 😄